static/js/vue-cdr-access/src/components/displayWrapper.vue

```diff
@@ -208,15 +208,20 @@ Top level component for full record pages with searching/browsing, including Adm
                return get(`${link}json`).then((response) => {
                    this.emptyJsonResponseCheck(response);
                    this.container_info = response.data;
-                   this.pageEvent(response.data);
+                   // Matomo
+                   this.matomoPageView(`Digital Collections Repository - ${this.container_info.pageSubtitle}`);
+                  // this.matomoPageEvent(this.container_info);
+
+                   // GA
                    this.pageView(this.container_info.pageSubtitle)
                    if (this.needsSearchResults) {
                        this.adjustFacetsForRetrieval();
                    } else {
                        this.is_page_loading = false;
                    }
                 }).catch(error => {
-                   this.setErrorResponse(error);
```

Comment:
Is this a change we want to keep or was it commented out for debugging?

Reply:
I've uncommented it. There was an error from matomo that was being eaten by this.